pkg/osvscanner/osvscanner.go

```diff
@@ -82,6 +83,25 @@ func scanDir(r *output.Reporter, query *osv.BatchedQuery, dir string, skipGit bo
 	})
 }
 
+func scanApkInstalledfile(r *output.Reporter, query *osv.BatchedQuery, path string) error {
+	parsedLockfile, err := lockfile.ParseApkInstalled(path)
+
+	if err != nil {
+		return err
```

Comment:
How clear is the error output if osv-scanner can't find the "/lib/apk/db/installed" file?